### PR TITLE
Enable using system keyring for accesstoken/PAT with azuredevops

### PR DESCRIPTION
**Pull request by bratne**
Wanted to store the secret using bugwarrior vault list / set. Didn't work with azure dev ops. Fixed in this PR

### CODE REVIEW

File: bugwarrior/services/azuredevops.py [hunk 1]
```diff
@@ -88,6 +76,9 @@ def get_work_item(self, workitemid):
     def get_work_items_from_query(self, query):
         data = str({"query": query})
         resp = self.session.post(f"{self.base_url}/wiql", data=data, params=self.params)
+        if resp.status_code == 401:
+            log.critical(f"HTTP 401 - Error autrenticating! Please check your PAT in the configuration")
```
Comment (ryneeverett) [inline bugwarrior/services/azuredevops.py]: typo: "autrenticating"

**Review by ryneeverett — APPROVED** [review]
Looks good other than the typo and flake8's formatting complaints (causing the CI failure).

File: bugwarrior/services/azuredevops.py (hunk 1 above)
Comment (bratne) [reply bugwarrior/services/azuredevops.py]: Haha. Fixed it in the last commit :P

**Comment by bratne** [comment]
typo and flake8 compatibility fixed

**Review by ryneeverett — APPROVED** [review]

**Pull request merged**